bin/cleanupDataOnTmp.sh

```diff
@@ -190,16 +191,27 @@ else
 			if [[ $(((dateInSecNow - dateInSecAnalysisData) / 86400)) -gt "${daysTillRemoval}" ]]
 			then
 				log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "Deleting ${projectName} on tmp because the project data is on prm for at least ${daysTillRemoval} days"
-				rm -Rfv "${TMP_ROOT_DIR}/"{projects,tmp,generatedscripts}"/${pipeline}/${projectName}/"
+				rm -Rfv "${TMP_ROOT_DIR}/"{projects,generatedscripts}"/${pipeline}/${projectName}/"
 				rm -vf "${TMP_ROOT_DIR}/logs/${projectName}/run01.projectDataCopiedToPrm.finished"
 			else
-				log4Bash 'TRACE' "${LINENO}" "${FUNCNAME:-main}" '0' " the projectDataCopiedToPrm.finished is $(((dateInSecNow - dateInSecAnalysisData) / 86400)) day(s) old. To remove the project, tmp and generatedscripts folders the ${TMP_ROOT_DIR}/logs/${projectName}/run01.projectDataCopiedToPrm.finished file needs to be at least ${daysTillRemoval} days old"
+				log4Bash 'TRACE' "${LINENO}" "${FUNCNAME:-main}" '0' " the projectDataCopiedToPrm.finished is $(((dateInSecNow - dateInSecAnalysisData) / 86400)) day(s) old. To remove the project and generatedscripts folders the ${TMP_ROOT_DIR}/logs/${projectName}/run01.projectDataCopiedToPrm.finished file needs to be at least ${daysTillRemoval} days old"
 				continue
 			fi
 		else
 			log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "${TMP_ROOT_DIR}/logs/${projectName}/run01.projectDataCopiedToPrm.finished does not exist, skipping"
 			continue
 		fi
+		#
+		# Removing tmp folder after 3 days 
+		#
+		if [[ $(((dateInSecNow - dateInSecAnalysisData) / 86400)) -gt "${daysTillRemovalTmpFolder}" ]]
+		then
+			log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "Deleting tmp/${pipeline}/${projectName} on tmp because the project data is on prm for at least ${daysTillRemovalTmpFolder} days"
```

Comment:
```suggestion
			log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "Deleting tmp/${pipeline}/${projectName} on tmp because the project data is on prm for at least ${daysTillRemovalTmpFolder} days."
```